yarn.lock

```diff
@@ -522,6 +522,13 @@
   resolved "https://registry.yarnpkg.com/@types/events/-/events-3.0.0.tgz#2862f3f58a9a7f7c3e78d79f130dd4d71c25c2a7"
```

Comment:
Is this a file we should really be checking in?

Reply:
Yes, it should be commited!